docs/develop/07_dApp/02_well-known-did-config.md

```diff
@@ -48,11 +48,11 @@ Make sure to create the DID with an `assertionMethodKey` so that you are able to
 ### Making the claim
 
 After you get a DID, you can make a claim about that DID.
-The claim has to be based on the [Domain Linkage CType][CType-Domain-Linkage]:
+The claim has to be based on the [Domain Linkage CType][CType-Domain-Linkage], whose definition you can get from the linked GitHub repository, or fetch from the blockchain using the CType's id:
 
```

Comment:
The id is missing.

Reply:
The id actually can't be there, so this is not an oversight. See https://github.com/KILTprotocol/ticket/issues/2732

Reply:
Uhmm. not happy, but ok

Reply:
I meant putting the id on the explanatory text.